hack/update-vanity-imports.sh

```diff
@@ -24,9 +24,8 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-kube::golang::verify_go_version
+kube::golang::internal::verify_go_version
```

Comment:
This is internal because you're not supposed to call it directly, it's supposed to be part of ::setup_env which now always verifies afterwards.

Reply:
see https://github.com/kubernetes/kubernetes/commit/d61f2996140e91950b3ad4c7e53db8de99c1833d